src/Microsoft.AspNet.Authentication.Google/GoogleHandler.cs

```diff
@@ -31,12 +31,8 @@ public GoogleHandler(HttpClient httpClient)
             response.EnsureSuccessStatusCode();
 
             var payload = JObject.Parse(await response.Content.ReadAsStringAsync());
-            
-            var context = new OAuthCreatingTicketContext(Context, Options, Backchannel, tokens, payload)
-            {
-                Properties = properties,
-                Principal = new ClaimsPrincipal(identity)
-            };
+
+            var context = new OAuthCreatingTicketContext(new ClaimsPrincipal(identity), properties, Context, Options, Backchannel, tokens, payload);
```

Comment:
I'll do that in a followup PR then, as its basically moving the creation of the ticket from the method CreateTicketAsync into whatever is provided in the context